Core/Core/Extensions/UIViewExtensions.swift

```diff
@@ -84,6 +84,11 @@ extension UIView {
         }
     }
 
+    public func pinToAllSidesOfSuperview() {
```

Comment:
Wouldn't `pin(inside: superview)` work for this?

Reply:
yeah, i ran into some bugs with changing orientation on ipad, i was trying different things with autolayout.  Will remove.